docs/src/pages/premium-themes/instapaper/theme/instapaper/components/tabs.js

```diff
@@ -15,52 +15,30 @@ export default ({ attach, theme, nest, ICON, TAB }) => ({
   },
   MuiTab: {
     root: {
-      lineHeight: 'inherit',
+      minHeight: 54,
+      fontWeight: 600,
       minWidth: 0,
-      marginRight: theme.spacing(1),
-      marginLeft: theme.spacing(1),
       [theme.breakpoints.up('md')]: {
         minWidth: 0,
-        marginRight: 30,
-        marginLeft: 30,
-      },
-      [attach(TAB.selected)]: {
```

Comment:
Hi @siriwatknp, If you have some time, I think that it would be great that we take the time to find the best possible theme implementation. We have different theme demos, the approach isn't consistent. I think that we should try to unify them.

Reply:
Hi @oliviertassinari, should we open a discussion topic about mui-theming?